src/App.js

```diff
@@ -1,33 +1,10 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          {' '}
-          {' '}
-          {' '}
-          Edit
-          {' '}
-          <code> Src / App.js </code>
-          {' '}
-          {' '}
-          {' '}
-          and save to reload
-          {' '}
-        </p>
-        {' '}
-        <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-          {' '}
-        </a>
-        {' '}
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends Component {
+  render() {
+    return <div className="app" />;
+  }
 }
```

Comment:
The `Calculator` component should be rendered in the `App` component. In the future, you will mostly route all parent components in the `App` component. The `App` component serves as the main structure for the application that contains the Navigation and different pages hosted by the application.

You should have something like this:
```
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```